.github/workflows/daily-smoke-test.yml

```diff
@@ -4,25 +4,25 @@ on:
   schedule:
     # Cron syntax is "minute[0-59] hour[0-23] date[1-31] month[1-12] day[0-6]". '*' is 'any value', and multiple values
     # can be specified with comma-separated lists. All times are UTC.
-    # So this expression means "run at 13:30 UTC every day". This time was chosen because it corresponds to
-    # 8:30AM CDT, meaning that any issues will be surfaced towards the start of business.
-    - cron: "30 13 * * *"
+    # So this expression means "run at 17:30 UTC every day". This time was chosen because it corresponds to
+    # 11:30AM CDT, meaning that any issues will be surfaced towards the start of business.
+    - cron: "30 17 * * *"
 
 jobs:
   # Step 1: Build the scanner tarball so it can be installed locally.
   build-scanner-tarball:
     name: 'Build scanner tarball'
     uses: ./.github/workflows/build-scanner-tarball.yml
     with:
-      target-branch: 'dev'
+      target-branch: 'dev-4'
```

Comment:
I'm confused. why are you chaging this to dev-4... this is the sfdx-code-analyzer-vscode repo... which doesn't have a dev-4.

The update should have been in the build-scanner-tarball.yml file on line 28 where 
```
git clone https://github.com/forcedotcom/sfdx-scanner.git sfdx-scanner
```
should now be
```
git clone -b dev-4  https://github.com/forcedotcom/sfdx-scanner.git sfdx-scanner
```